### DISCUSSION TITLE
Renovate close PRs opened by another Renovate run

### DISCUSSION THREAD

**Discussion by julien-michaud** [renovatebot/renovate#29093] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github - renovatebot/github-action@23b01dbf6f38965076a36aff87f802263164f204

### Please tell us more about your question or problem

I have multiple reusable workflows located in multiples repos 

- repo1 (reusable-workflow-1a, reusable-workflow-1b, reusable-workflow-1c)
- repo2 (reusable-workflow-2a, reusable-workflow-2b, reusable-workflow-2c)

Renovate is launched in both repos, the repos 1 & 2 are tasked to update the various repos using the reusable workflows coming from them. 

Here is the conf from the repo1: 

```
{
  "enabledManagers": [
    "github-actions"
  ],
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "enabled": false
    },
    {
      "matchPackageNames": ["repo1"],
      "versionCompatibility": "^(?<compatibility>.*)/(?<version>.*)$",
      "prBodyNotes": ["{{#if isMajor}}:warning: Please make sure to read the migration guide corresponding to the new version before merging :warning:{{/if}}"],
      "prBodyTemplate": "{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{changelogs}}}{{{footer}}}",
      "enabled": true
    }
  ]
}
```

conf from the repo2

```
{
  "enabledManagers": [
    "github-actions"
  ],
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "enabled": false
    },
    {
      "matchPackageNames": ["repo2"],
      "versionCompatibility": "^(?<compatibility>.*)/(?<version>.*)$",
      "prBodyNotes": ["{{#if isMajor}}:warning: Please make sure to read the migration guide corresponding to the new version before merging :warning:{{/if}}"],
      "prBodyTemplate": "{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{changelogs}}}{{{footer}}}",
      "enabled": true
    }
  ]
}
```

When the Renovate from repo1 is finished, PRs are created on various repos to update their reusable workflows. 
But when the Renovate from repo2 is finished, it closed the PRs opened from the repo1 Renovate and only keep the PRs opened to update the reusable workflows coming from repo2. 

Is their a way to keep the Renovate PRs opened ? 

Or do I need to launch Renovate from a central place with the following conf: 

```
{
  "enabledManagers": [
    "github-actions"
  ],
  "packageRules": [
    {
      "matchPackagePatterns": ["*"],
      "enabled": false
    },
    {
      "matchPackageNames": ["repo1"],
      "versionCompatibility": "^(?<compatibility>.*)/(?<version>.*)$",
      "prBodyNotes": ["{{#if isMajor}}:warning: Please make sure to read the migration guide corresponding to the new version before merging :warning:{{/if}}"],
      "prBodyTemplate": "{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{changelogs}}}{{{footer}}}",
      "enabled": true
    },
    {
      "matchPackageNames": ["repo2"],
      "versionCompatibility": "^(?<compatibility>.*)/(?<version>.*)$",
      "prBodyNotes": ["{{#if isMajor}}:warning: Please make sure to read the migration guide corresponding to the new version before merging :warning:{{/if}}"],
      "prBodyTemplate": "{{{header}}}{{{table}}}{{{warnings}}}{{{notes}}}{{{changelogs}}}{{{footer}}}",
      "enabled": true
    }
  ]
}
```

Thanks ! 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Having two bots on the same repo is generally not recommended for reasons like this. If you do, you want them to have different identities and branch prefixes, so that they stay separate.

But if you are doing it then set pruneStaleBranches=false. It means that Renovate can't/won't autoclose PRs so you need to do that manually
